### PR TITLE
Add laravel auth

### PR DESCRIPTION
**Pull request by vuminhhieu1311**
## Related Tickets
- [#39331](https://edu-redmine.sun-asterisk.vn/issues/39331)
## WHAT (optional)
- Login, Logout
## WHY (optional)
- Using laravel/ui: https://github.com/laravel/ui
## Evidence (Screenshot or Video)
Screen Shot 2021-08-03 at 14 09 02

### CODE REVIEW

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    <div class="col-md-8 col-md-offset-2">
+                        <!-- login_wrapper -->
+                        <h1>{{ __('Login') }}</h1>
+                        <form action="{{ route('login') }}" method="post">
+                            @csrf
+                            <div class="login_wrapper">
+                                <div class="formsix-pos">
+                                    <div class="form-group i-email">
+                                        <input type="email" class="form-control" id="email"
+                                            placeholder="{{ __('E-Mail Address') }}" @error('email') is-invalid @enderror"
```
Comment (nvvu99) [inline resources/views/auth/login.blade.php:18]: Thừa dâu nháy kép (") ở cuối dòng phải ko?

**Review by nvvu99 — COMMENTED** [review]
Dòng 18 file login.blade.php thừa dấu ".

File: resources/views/auth/login.blade.php (hunk 1 above)
Comment (vuminhhieu1311) [reply resources/views/auth/login.blade.php]: đã fix ạ

Comment (vuminhhieu1311) [reply resources/views/auth/login.blade.php]: A approve đi

**Review by nvvu99 — APPROVED** [review]
Ok

**Comment by vuminhhieu1311** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**